### PR TITLE
fix(core): improve component a11y

### PR DESCRIPTION
**Pull request by nuke-ellington**
## 💡 What is the current behavior?

Components provide lighthouse a11y scores of less than 100% because of misused semantic html and aria attributes

GitHub Issue Number: IX-1749

## 🆕 What is the new behavior?

- affected components should have 100% a11y scores
-

## 🏁 Checklist

A pull request can only be merged if all of these conditions are met (where applicable):

- [ ] 🦮 Accessibility (a11y) features were implemented
- [ ] 🗺️ Internationalization (i18n) - no hard coded strings
- [ ] 📲 Responsiveness - components handle viewport changes and content overflow gracefully
- [ ] 📄 Documentation was reviewed/updated (`pnpm run docs`)
- [x] 🧪 Unit tests were added/updated and pass (`pnpm test`)
- [x] 📸 Visual regression tests were added/updated and pass ([Guide](https://github.com/siemens/ix/blob/main/CONTRIBUTING.md#visual-regression-testing))
- [x] 🧐 Static code analysis passes (`pnpm lint`)
- [x] 🏗️ Successful compilation (`pnpm build`, changes pushed)

### CODE REVIEW

**Review by matthiashader — CHANGES_REQUESTED** [review]
changeset missing

File: packages/core/src/components/menu/menu.tsx [hunk 1]
```diff
@@ -599,7 +599,7 @@ export class Menu {
         }}
         slot="menu"
       >
-        <aside
+        <nav
```
Comment (matthiashader) [inline packages/core/src/components/menu/menu.tsx:602]: Is this change necessary? Should the aside stay and the <div id="menu-tabs" be replaced with nav? (https://www.w3.org/TR/2011/WD-html5-author-20110809/the-aside-element.html, https://stackoverflow.com/questions/40390390/semantic-differences-between-nav-and-aside-for-navigation-in-html)

File: packages/core/src/components/pane/pane.tsx [hunk 2]
```diff
@@ -149,10 +149,10 @@ export class Pane {
   @State() private parentWidthPx = 0;
   @State() private parentHeightPx = 0;
 
-  private validPositions = ['top', 'left', 'bottom', 'right'];
-  private collapsedPane = '40px';
-  private collapsedPaneMobile = '48px';
-  private animations: Map<string, anime.AnimeInstance> = new Map();
+  private readonly validPositions = ['top', 'left', 'bottom', 'right'];
+  private readonly collapsedPane = '40px';
+  private readonly collapsedPaneMobile = '48px';
```
Comment (matthiashader) [inline packages/core/src/components/pane/pane.tsx:154]: `private static readonly collapsedPaneMobile`

**Review by matthiashader — CHANGES_REQUESTED** [review]

**Comment by matthiashader** [comment]
Further elements that haven't 100% accessibility score in Lighthouse:
- ix-push-card
- ix-breadcrumb
- ix-icon-toggle-button
- ix-slider

**Comment by nuke-ellington** [comment]
> changeset missing
> 
> reduce strict errors

Category filter strict errors will be addressed via #1522

File: packages/core/src/components/menu/menu.tsx [hunk 3]
```diff
@@ -620,11 +607,15 @@ export class Menu {
...
             )}
             {this.breakpoint === 'sm' &&
-              this.applicationLayoutContext.appSwitchConfig && (
+              this.applicationLayoutContext?.appSwitchConfig && (
                 <ix-icon-button
-                  onClick={() =>
-                    showAppSwitch(this.applicationLayoutContext.appSwitchConfig)
-                  }
+                  onClick={() => {
+                    if (this.applicationLayoutContext?.appSwitchConfig) {
```
Comment (matthiashader) [inline packages/core/src/components/menu/menu.tsx:613]: Why do we need this query `if (this.applicationLayoutContext?.appSwitchConfig) {`. Shouldn't this be covered with line 610?

**Review by matthiashader — APPROVED** [review]

**Pull request merged**